internal/karma/karma.conf.js

```diff
@@ -39,19 +39,21 @@ if (process.env['WEB_TEST_METADATA']) {
     // "@io_bazel_rules_webtesting//browsers:firefox-local"
     // then the 'environment' will equal 'local' and
     // 'webTestFiles' will contain the path to the binary to use
-    const webTestNamedFiles = webTestMetadata['webTestFiles'][0]['namedFiles'];
-    if (webTestNamedFiles['CHROMIUM']) {
-      // When karma is configured to use Chrome it will look for a CHROME_BIN
-      // environment variable.
-      process.env.CHROME_BIN = require.resolve(webTestNamedFiles['CHROMIUM']);
-      browsers.push(process.env['DISPLAY'] ? 'Chrome': 'ChromeHeadless');
-    }
-    if (webTestNamedFiles['FIREFOX']) {
-      // When karma is configured to use Firefox it will look for a FIREFOX_BIN
-      // environment variable.
-      process.env.FIREFOX_BIN = require.resolve(webTestNamedFiles['FIREFOX']);
-      browsers.push(process.env['DISPLAY'] ? 'Firefox': 'FirefoxHeadless');
-    }
+    webTestMetadata['webTestFiles'].forEach(webTestFiles => {
```

Comment:
why did we look at only the first item before and now we look at all of them?
/cc @vikerman

Reply:
After updating to the latest rules_webtesting (Filipe’s Windows year coverage change), FIREFOX_BIN is no longer the first entry in the list.

Also, Firefox tests are broken as well after Filipe’s commit. Not sure how it passed CI in that PR.

Reply:
After updating to the latest rules_webtesting (Filipe’s Windows year coverage change), FIREFOX_BIN is no longer the first entry in the list.

Also, Firefox tests are broken as well after Filipe’s commit. Not sure how it passed CI in that PR.